internal/batches/service/service.go

```diff
@@ -43,10 +43,12 @@ func New(opts *Opts) *Service {
 // The reason we ask for batchChanges here is to surface errors about trying to use batch
 // changes in an unsupported environment sooner, since the version check is typically the
 // first thing we do.
-const sourcegraphVersionQuery = `query SourcegraphVersion {
+
+const getInstanceInfo = `query SourcegraphVersion {
```

Comment:
Should probably match the query name:

```suggestion
const getInstanceInfo = `query InstanceInfo {
```